### PR TITLE
Connect scan utils

### PR DESCRIPTION
**Pull request by kssampson**

### CODE REVIEW

File: src/main.jsx [hunk 1]
```diff
@@ -29,6 +30,10 @@ const router = createBrowserRouter([
     path: "/scanner",
     element: <Scanner />
   },
+  {
+    path: "/reservationDetails",
```
Comment (kjmak14) [inline src/main.jsx:34]: Right now I have a path with  ':reservation_id' at the end of this url. Try to see if you can incorporate that into this path or we can consolidate on how you want to do that later

**Review by kjmak14 — APPROVED** [review]
Looks good. We can hash out how we want to incorporate the pathing to the reservation details page

File: src/components/valet-reservation/ReservationDetails.jsx
Comment (stevkim) [inline src/components/valet-reservation/ReservationDetails.jsx:1]: This page will be refactored later upon flow finalization

**Review by stevkim — APPROVED** [review]
Everything looks great, nice progress.

-- For future, make sure to remove console.logs, thanks!

**Pull request merged**